### PR TITLE
Update index.md

### LINKED ISSUE #30281: Remove note
Just above [Log Level](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?branch=main&branchFallbackFrom=pr-en-us-30249&view=aspnetcore-7.0#log-level)

See [this comment](https://github.com/dotnet/AspNetCore.Docs/pull/30249#discussion_r1319327121)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 7184a308-a254-9350-a5cd-5bced1f369ae
* Version Independent ID: 726e3bf1-f367-d733-8933-bccc04da0e16
* Content: [Logging in .NET Core and ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?branch=main&branchFallbackFrom=pr-en-us-30249&view=aspnetcore-7.0)
* Content Source: [aspnetcore/fundamentals/logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/logging/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #30281

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/30f0ddb1f0725467663a738785c0118424812372/aspnetcore/fundamentals/logging/index.md) | [Logging in .NET Core and ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/index?branch=pr-en-us-30282) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@cijothomas what about moving it to the very bottom of the file under it's own H2? Doesn't the comment still apply?

If not, happy to remove the note.

**Comment by cijothomas** [comment]
I'd suggest removing it now.

(once we decide to show how to do the logging even before DI/host is ready, then we need to show LoggerFactory.Create to create a alternate pipeline. When we add that, this Note can be brought back.)

**Review by tdykstra — APPROVED** [review]

File: aspnetcore/fundamentals/logging/index.md [hunk 1]
```diff
@@ -887,6 +885,10 @@ If the `traceparent` http request header is set, the `ParentId` in the log scope
 
 To create a custom logger, see [Implement a custom logging provider in .NET](/dotnet/core/extensions/custom-logging-provider).
 
+## LoggerFactory.Create
+
+***NOTE:*** [LoggerFactory.Create](/dotnet/api/microsoft.extensions.logging.loggerfactory.create) Creates a new instance of <xref:Microsoft.Extensions.Logging.ILoggerFactory> that's configured using the provided `configure` delegate. The new instance of `ILoggerFactory` creates a new logging pipeline, different from the default logging pipeline. The new logging pipeline is not configured by the `Logging` section of `appsettings.json` or `appsettings.{ENVIRONMENT}.json`. For more information, see [Logging configuration](xref:fundamentals/logging/index#configure-logging).
+
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/logging/index.md:891]:
```suggestion
<!-- See https://github.com/dotnet/AspNetCore.Docs/pull/30282#issuecomment-1712135965
## LoggerFactory.Create

***NOTE:*** [LoggerFactory.Create](/dotnet/api/microsoft.extensions.logging.loggerfactory.create) Creates a new instance of <xref:Microsoft.Extensions.Logging.ILoggerFactory> that's configured using the provided `configure` delegate. The new instance of `ILoggerFactory` creates a new logging pipeline, different from the default logging pipeline. The new logging pipeline is not configured by the `Logging` section of `appsettings.json` or `appsettings.{ENVIRONMENT}.json`. For more information, see [Logging configuration](xref:fundamentals/logging/index#configure-logging).

```
-->

**Comment by Rick-Anderson** [comment]
Moved to #30288

**Pull request closed without merging**